app/(shop)/[...category]/utils.ts

```diff
@@ -0,0 +1,135 @@
+import { NumberFilter, StringFilter, TenantFilter } from '@/generated/discovery/graphql';
+import { PRICE_FIELD } from './constants';
+import { FilterOption } from '@/app/(shop)/[...category]/types';
```

Comment:
Isn't it this just `./types`?

Reply:
Auto imports from webstorm... It's solved now